examples/castor/amqp_upload.py

```diff
@@ -195,19 +195,19 @@ def main():
                         required=True, help='data directory')
     parser.add_argument('--pattern', action='store', dest='pattern',
                         required=True, help='file filter')
-    parser.add_argument('--batch', action='store', dest='batch',
-                        required=True, help='batch x messages')
```

Comment:
I think its better to leave this as before. But change "required" to False, and add a default to the number you want.